package.json

```diff
@@ -12,7 +12,8 @@
     "build-component": "mkdir dist || rm -r dist && rollup --config",
     "build-site": "mkdir dist || rm -r dist && node_modules/.bin/webpack --config webpack.site.prod.js --mode production",
     "test": "browserify tests/*.spec.js -p esmify | tape-run",
-    "start": "node_modules/.bin/webpack-dev-server --mode development --open --hot"
+    "start": "node_modules/.bin/webpack-dev-server --mode development --open --hot",
+    "postversion": "./publish.sh && git push && git push --tags"
```

Comment:
I think actually publishing is more than what people will expect `npm version` by itself to do:

https://docs.npmjs.com/cli/version:
> Run the postversion script. Use it to clean up the file system or automatically push the commit and/or tag.